packages/agreement-process/src/services/agreementService.ts

```diff
@@ -418,7 +418,6 @@ export function agreementServiceBuilder(
 
       const agreements = (
         await readModelService.getAllAgreements({
-          producerId: agreement.data.producerId,
```

Comment:
Unrelated, but reviewing the submission logic I found this difference with the Scala code
https://github.com/pagopa/interop-be-agreement-process/blob/88402a001d4c285fb45da4081d782fb696a2894a/src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala#L777-L778